inventory/first.yml

```diff
@@ -11,3 +11,12 @@ all:
     etcdctl_client_key: "LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFb3dJQkFBS0NBUUVBd0RCL2o4U0lydlZaeUVzREprN1FsaEtEOFJKSjFzYWMwNitCTnI2UmJUME9XSzlvCis2enpVekVxT1FZeUt5L1lvcC9vT3RSMDBxcFBQMFp3c3lJaFVocmIxYlppNVVOYWpBNFZLbU4rbHlrRitjK3oKU0tqZnM0bEJVK3d1c0ZiU0xyNVNEWXZKNGw5bW40dzlrOVU5aE1BemhmaVlsRUpzT2E4ZG1aeitPeTBmSWRwUwprQlA1YVhLb29qYXQ5SmFGMVd6cnkvZ2V1d2hscEJLNzFpaGIxcDFJTDRjWnV6VURqamhmQ0Q3UG5VYUNhUENjClU3NzlUeHY0aTN6MXJsbGdCQ0d4V0tYMUZ1ajZXTXNGcjg2cW5yUVF4OUtoUnFGYkd4SGFxUCsrMWZCRFZqMjYKZVNyMnFYQU1HcWRTQmRkN3VIY0VIbjB6OGtzSWpxZXhDUjBQQndJREFRQUJBb0lCQVFDVGxHcFA1dFlpcTRyZgpFQ01DQnFHTHEvb2ZZekRLT1VkeUVpSUIvTUpGaFE0QU1lcEc0SjhVc2ZzL1FLL0RsQUVobC9OSFFaL1hxU29qClo1N2hINE5BdmhremVkTmxRNXhEVWdrVXZNUVhwVXZPY1p5MVhlQXh5R01lNDQxU29jalNWaGxDQjZibUt1WHQKL0UzSXVJVjk2KzFrb2VuZ1owVWQ0b2V6VHVlMlhmSDlwV3hDa09qUzJWdWJNN3BNMU5qc2x5UnZualAvc0I2cApMQzRHWlZZb1Y3cUJZMTlPcWtyZitQR3N2WGpqVGo1UFNJN0RDczZ5bWM4bm41eHllREJ4TzBCMVUvWjJRODI4CkVNUTVnNHFYKzlNQnFRZThhQS9xbGg0RnlaNGl6eVlQWjNVUlYvSmpXWDBweXlLd1ViT0cyQUJKYWZwQXZpaEcKV0pFRTRqbFJBb0dCQU5qL1dmeHpSZDc4WTdQczl2UWtlYjRJaTFsb3h4MVd5N2F3SkI1MjFKdG1jeURWVUxuNgpJK1lGSmVzSStyQXBaOUF4SnQ3V0EwN1ZBTTlmVnJoSkJ6cmZjSW55Wng3WEdEZVoyYWEwcUkvNmtlRTdnNHMvCnZNZVluUHdGWTZzL3NUWjhvNHFvRStZbnZLTlNmcmtOakM5bVd2OHBuOFlTL3FQY3B2MEFpWUwvQW9HQkFPSzcKcVdkWGp1NkRBS1lzVFVaZUFHV0lwUWcyVjMxenlEaXFOVHhUYzlFaWxac1crVko3YnhBdm0reWEyV2s1ZDFUago3TzV2UUxpekc1TDVNZHBhVGU3OW90WDBZZWhGYU9WTUNCS1JtemNHQ05rRzYweDRYUEpiZHdub3Q2OEd0UERKCnBGYUJBRlpoR0J6YzdwNkpxL0lTRm5MUklnc2dTQ3ZNN0hINk5GdjVBb0dBYXBFVTV6YlJOSTRzaExzMkdIWWwKVjU4b21ETXR2Z25YSjVQZldScXdsTlMrYTN4M0ZhNXpuTTRnYmduTjIxK3FQUU0yZks4TGx0WmlmSFRzeUxvTQprK3dSZjUyTTZRZG5QRDhSTlo1NVBhUW5xajRvL3VUQldDQnAwQmJ4ZUxOaDIwbGdvNEtocXlFT3NnS2NlMUxjCjZwWlEwQWpvRDBWYXlxeGRVZGpHcjQ4Q2dZQTE2Kzl6MWt6MXlFNEZOeEFCcGlWdUFEdFBZcTBYOXNzdFByMSsKWlZKcjhFK29IS1NheXlRMHN6M2YzVXZMVEpGbXJQWVNGbjlMN2J5bkRZQ2ZyMDVOZ2lDUnhaYnJTR2FUOGFzUgp6M01jZU9EdHpFbDFYRUt1bUxZTXN6M0w4RXBqNFhDbDZmUU1sdmJIRm52aFZGaUNMMjhMUzFWUFZCUVY1cHhzCjF0WlE4UUtCZ0d5ODVMZmp3L09INS9Xc0oxbVptYkFCWm9CcW1xU2pTdk9RWmJHSFd3aWh6RnBUMXUyWEtGYjUKNTNzLytvdEJ1dENlYTQ1OGRia0ZsdkVvM2VxVVp2WlhjOEp3eFVMb0ZZUndKaEFDQ1VTWllUOXJSckFudVRWMQpIRXdTWUdOK1VzL1BQeDNCQ3dSL1F4T3FmMFk3RHhqOE44OHo0SnY2R1ppazdNdm5WaEsvCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg=="
+
```

Comment:
I added logic to create these group automagically. Can you rebase again?

Reply:
This is required because we can't define an empty group from playbooks